package.json

```diff
@@ -30,11 +30,12 @@
     "marked": "^0.3.5",
     "mime": "^1.3.4",
     "mime-types": "^2.1.6",
-    "panda-9000": "2.0.0-alpha-02",
+    "panda-9000": "pandastrike/panda-9000#2326eacfddc6e8df5eb5d6000bb4d0c0dfd8033c",
```

Comment:
TODO: Fix when P9K https://github.com/pandastrike/panda-9000/issues/15 is fixed and deployed to NPM